File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex19.c
#include <stdio.h>

main()
{
    //vars
    float largura, profundidade, area, potencia;

    printf("\t\t\tPotencia de iluminacao\n\n");

    //Entrada
    printf("Insira a largura do comodo em metros: ");
    scanf("%f",&largura);

    printf("Insira a profundidade do comodo em metros: ");
    scanf("%f",&profundidade);

    //Processamento
    area = largura * profundidade;
    potencia = area * 18;

    //Saída
    printf("Area do comodo: %.2f\n", area);
    printf("Potência para iluminar o comodo: %.2f\n", potencia);



}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex19.java
//Importando Scanner
import java.util.Scanner;

public class Ex19 {
	
	public static void main(String[] args)
	{
	//	Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	float largura, profundidade, area, potencia;
	
	System.out.println("Potência para iluminar um cômodo\n\n");
	
	//Entrada	
	System.out.print("Insira a largura do cômodo em metros: ");
	largura = input.nextFloat();
	
	System.out.print("Insira a profundidade do cômodo em metros:");
	profundidade = input.nextFloat();
	
	//Processamento
	area = largura * profundidade;
	potencia = area * 18;
	
	//Saída
	System.out.println("Área = " + area + " m²");
	System.out.println("Potência = " + potencia + " W ");
	
	
	}
}
